src/pagespeed.js

```diff
@@ -1,13 +1,22 @@
 const axios = require('axios');
 
-module.exports = async function(page, apiKey) {
-	let url = `https://www.googleapis.com/pagespeedonline/v5/runPagespeed?url=${page}&key=${apiKey}&category=PERFORMANCE&category=ACCESSIBILITY`;
+const strategies = {
+	DESKTOP: 'DESKTOP',
+	MOBILE: 'MOBILE'
+};
+
+const getPagespeedInsights = async (page, apiKey, strategy) => {
+	let url = `https://www.googleapis.com/pagespeedonline/v5/runPagespeed?url=${page}&key=${apiKey}&category=PERFORMANCE&category=ACCESSIBILITY&strategy=${strategy}`;
```

Comment:
Check for a valid `strategy` and if not throw an exception?
I should have done that for the other args too.